web/cypress/support/commands.js

```diff
@@ -66,6 +66,18 @@ Cypress.Commands.add('configurePlugins', (cases) => {
 
         // NOTE: wait for the Drawer to appear on the DOM
         cy.focused(domSelector.drawer).should('exist');
+
+        cy.get(domSelector.codeMirrorMode).invoke('text').then(text => {
+          if (text === 'Form') {
+            // FIXME: https://github.com/cypress-io/cypress/issues/7306
```

Comment:
we need an issue instead of a comment.

Reply:
Refer issue: https://github.com/apache/apisix-dashboard/issues/1738

Reply:
so please remove this comment

Reply:
ok